### ISSUE TITLE
Update configuration reference "System" for Adobe Stock Integration

### ISSUE THREAD

**Issue by filmaj** [magento/merchdocs#89]
# Topic clarification/correction request

## Topic Link

https://docs.magento.com/m2/ee/user_guide/configuration/advanced/system.html
From: `src/configuration/advanced/system.md`

## Description

- [ ] After the "Image Upload Configuration" section, add new "Adobe Stock Integration" section
- [ ] Include screen of new fields that matches others in the topic in terms of size and positioning within the Admin UI
- [ ] Include a table that describes each field

## Additional information

This should be assigned to the [Adobe Stock Integration Docs Project](https://github.com/magento/merchdocs/projects/6) and stems from #77. I am happy to take this one.

**Comment by filmaj** [comment]
Once  the `adobe-stock-images` branch is merged into the `adobe-stock` branch, I think we can close this issue.

**Comment by filmaj** [comment]
It  seems like `adobe-stock-images` is already merged into `adobe-stock`, so closing this as it is ready for final review.
